### ISSUE TITLE
Problem running OpenShift 3.11

### ISSUE THREAD

**Issue by joe-eklund** [causefx/Organizr#1557]
###### Operating System: OpenShift 3.11
<hr>

##### Problem Description:

I am trying to deploy Organizr on OpenShift 3.11 and see an error that says `Webserver not setup for Organizr v2.1`. I am using OpenShift S2I (source to image) to build and deploy it in a pod. I point to this repo and it builds everything fine, then deploys it. But I get the error shown below when I navigate to the domain.

<hr>

##### Reproduction Steps:

Requires you to have an OpenShift cluster, but to reproduce I just did a PHP 7.2 deploy, pointed it to this repo, selected v2-master, and off it went. Then setup a route and it worked (excepted for the error described).

<hr>

#### Errors on screen?  If so paste here:
<img width="558" alt="Screen Shot 2021-01-07 at 3 44 50 PM" src="https://user-images.githubusercontent.com/8761687/103957652-08895580-5100-11eb-9c36-99a37af23bb8.png">

```
Errors in Chrome console:

/plugins/bower_components/owl.carousel/owl.video.play.png:1 Failed to load resource: the server responded with a status of 404 (Not Found)
functions.js?v=2.1.120:10352 https://docs.organizr.app/books/setup-features/page/organizr-20--%3E-21-migration-guide
functions.js?v=2.1.120:10321  API V2 API  If you see a 404 Error for api/v2/launch below this line, you have not setup the new location block... See URL above this line 
/api/v2/launch:1 Failed to load resource: the server responded with a status of 404 (Not Found)
DevTools failed to load SourceMap: Could not load content for https://my_domain/js/tinycolor.min.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE
DevTools failed to load SourceMap: Could not load content for https://my_domain/js/plyr.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE

I do see similar errors on the server logs:

10.128.0.1 - - [07/Jan/2021:15:46:36 -0800] "GET /api/v2/launch HTTP/1.1" 404 196 "https://my_domain.com/ " "Mozilla/5.0 (Macintosh; Intel Mac OS X 10_15_7) AppleWebKit/537.36 (KHTML, like Gecko)"
```

Any ideas?

**Comment by Roxedus** [comment]
2.1 require additions to the webserver config.

**Comment by joe-eklund** [comment]
If my webserver is setup to forward all requests of a particular domain, what else do I need to setup? I am new to OpenShift and still learning. I am trying to setup persistent volumes, and am wondering if that has anything to do with it. I'll let you know, but if anyone else has any thoughts on setting it up on OCP let me know. :)

**Comment by Roxedus** [comment]
The changes you need to do are described here https://docs.organizr.app/books/setup-features/page/organizr-20--%3E-21-migration-guide

**Comment by joe-eklund** [comment]
Ah so it says for docker users it should already be included, but since I am doing a source2image build in OpenShift, I think I will need to replicate those changes. I took a look at https://github.com/Organizr/docker-organizr/blob/master/root/defaults/default and it looks like this is the issue I am having. I see the new api stuff listed in there, which is obviously not included in the base php repo.

So instead of doing a S2I build of the original PHP repo (this one), maybe I can try to get a S2I build of the docker repo. I tried to run the docker image directly from docker hub but ran into issues since it required root access (and my OpenShift cluster did not like that).

Another idea I had was to spin up a reverse proxy pod that handles the needed changes. I will report back my findings when I get a chance.
